app-shells/radian/radian-0.6.9.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit distutils-r1
+
+DESCRIPTION="A 21 century R console"
+HOMEPAGE="https://github.com/randy3k/radian"
+SRC_URI="https://github.com/randy3k/radian/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
+	dev-python/prompt-toolkit[$PYTHON_USEDEP]
+	dev-python/rchitect[$PYTHON_USEDEP]
+	dev-python/six[$PYTHON_USEDEP]
+	dev-lang/R"
```

Comment:
These are probably `RDEPEND`s.

DEPEND is for things we need present during build
BDEPEND is for things we need to execute in order to build
RDEPEND is for things we need during runtime of what we have built

Reply:
They can indeed be `REPEND`s